types/wordpress__data-controls/index.d.ts

```diff
@@ -1,11 +1,10 @@
-// Type definitions for @wordpress/data-controls 2.2
+// Type definitions for @wordpress/data-controls 2.30
 // Project: https://github.com/WordPress/gutenberg/tree/master/packages/data-controls/README.md
 // Definitions by: Derek Sifford <https://github.com/dsifford>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.7
 
 import { APIFetchOptions } from '@wordpress/api-fetch';
-import { Action } from '@wordpress/data';
```

Comment:
This didn't really give us anything useful, so we might as well remove the dependency on wp data